### DISCUSSION TITLE
HelpProvider colors

### DISCUSSION THREAD

**Discussion by sebbarg** [spectreconsole/spectre.console#1357] [Q&A] [answered]
The colors in the [HelpProvider](https://github.com/spectreconsole/spectre.console/blob/main/src/Spectre.Console.Cli/Help/HelpProvider.cs) are hardcoded. Showing help on a bright background (e.g. old school default MacOS terminal) becomes really hard to read, primarily because of the yellow colors. 

How do I deal with that? I can create a custom help provider, but that seems a bit much just to fix some hardcoded colors.

**Comment by FrankRay78** [comment]
Probably a valid case for localising the colours, like what we've just done with the help text. Or maybe having them as config parameters on the help writer, or similar. Agree you shouldn't have to re-implement the entire help writer just for this. Plus hardcoded colours is bad for accessibility.

**Comment by FrankRay78** [comment, marked as answer]
I've created an issue for this: https://github.com/spectreconsole/spectre.console/issues/1379

**Comment by FrankRay78** [comment]
Check out this branch if you get a chance @sebbarg :  https://github.com/spectreconsole/spectre.console/pull/1408

**Reply by sebbarg** [reply]
Awesome!
